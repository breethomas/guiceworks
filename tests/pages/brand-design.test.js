import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'
import App from '../../pages/brand-design'

describe('/brand-design', () => {
  let app
  let text

  const getTitleText = () => (
    shallow(app)
    .children()
    .find('Title')
    .children()
    .text()
  )

  beforeEach(() => {
    app = (<App url={{ pathname: '/brand-design' }} />)
    text = getTitleText()
  })

  it('samples a text node from the DOM', () => {
    expect(text).toEqual('Brand Design')
  })

  it('shows the correct snapshot', () => {
    const component = renderer.create(app)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})

