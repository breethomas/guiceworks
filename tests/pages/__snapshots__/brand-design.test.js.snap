// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/brand-design shows the correct snapshot 1`] = `
<div>
  <nav
    className={
      Object {
        "data-css-14eak1y": "",
      }
    }
  >
    <div
      className={
        Object {
          "data-css-1cnlars": "",
        }
      }
    >
      <a
        className="css-4w26lw css-ej55tb"
        href="/"
        onClick={[Function]}
      >
        <svg
          height={24}
          width={24}
        >
          <g
            fill="currentColor"
          >
            <path
              d="M24,0 C24,0 23.4836048,9.80252996 16.4820958,15.2719707 L16.4128111,19.0346251 L11.4820194,24 L11.4746834,20.2882431 L9.63105705,18.6292449 C8.29671435,22.7691304 1.26439406,22.7691304 1.26439406,22.7691304 C1.26439406,22.7691304 1.26439406,16.3396736 5.58319762,14.4450082 L3.94220876,12.6079505 L-2.84217094e-14,12.6079505 L5.07440302,7.66843342 L8.7285621,7.58627726 C14.4472436,0.671411685 24,0 24,0 Z M16.425,10.1644246 C17.8333261,10.1644246 18.975,9.01830529 18.975,7.60449544 C18.975,6.19068562 17.8333261,5.0445663 16.425,5.0445663 C15.0166739,5.0445663 13.875,6.19068562 13.875,7.60449544 C13.875,9.01830529 15.0166739,10.1644246 16.425,10.1644246 Z"
            />
          </g>
        </svg>
      </a>
      <a
        className={
          Object {
            "data-css-4w26lw": "",
          }
        }
        href="/experience-design"
        onClick={[Function]}
      >
        Experience Design
      </a>
      <a
        className={
          Object {
            "data-css-4w26lw": "",
          }
        }
        href="/interface-design"
        onClick={[Function]}
      >
        Interface Design
      </a>
      <a
        className={
          Object {
            "data-css-wp101j": "",
          }
        }
        href="/brand-design"
        onClick={[Function]}
      >
        Brand Design
      </a>
      <a
        className={
          Object {
            "data-css-4w26lw": "",
          }
        }
        href="/graphic-design"
        onClick={[Function]}
      >
        Graphic Design
      </a>
      <a
        className={
          Object {
            "data-css-4w26lw": "",
          }
        }
        href="/front-end-engineering"
        onClick={[Function]}
      >
        Front-end Engineering
      </a>
    </div>
  </nav>
  <main>
    <section
      className={
        Object {
          "data-css-clr3te": "",
        }
      }
    >
      <h1
        className={
          Object {
            "data-css-1iq5a26": "",
          }
        }
      >
        Brand Design
      </h1>
    </section>
  </main>
</div>
`;
